fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsManager.java

```diff
@@ -58,6 +60,27 @@ public Statistics getStatistics() {
         return statistics;
     }
 
+    /**
+     * Support for deleting table or partition statistics.
+     *
+     * @param stmt get table name and partition name from it.
+     */
+    public void dropStats(DropTableStatsStmt stmt) {
```

Comment:
seems that the grammer doesn't allow to drop multiple table in single statement though